demisto_sdk/commands/validate/sdk_validation_config.toml

```diff
@@ -416,6 +416,7 @@ select = [
   "RM109",
   "RM113",
   "RM114",
+  "RM116",
```

Comment:
Do we want this error to be ignorable?
If so, let's make sure to add this under the ignorable section as well.